drivers/sensor/lsm303dlhc_accel/Kconfig

```diff
@@ -0,0 +1,79 @@
+# Kconfig - LSM303DLHC Three Axis Accelerometer configuration options
+
+#
+# Copyright (c) 2018 Philémon Jaermann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig LSM303DLHC_ACCEL
+	bool
+	prompt "LSM303DLHC Three Axis Accelerometer"
+	depends on I2C
+	default n
+	help
+	  Enable driver for LSM303DLHC I2C-based triaxial accelerometer sensor.
+
+config LSM303DLHC_ACCEL_NAME
```

Comment:
Remove `LSM303DLHC_ACCEL_NAME`, `LSM303DLHC_ACCEL_I2C_MASTER_DEV`, and `LSM303DLHC_ACCEL_I2C_ADDR`. These should come from device tree instead.